src/social-network-provider/facebook.com/tasks/uploadToPostBox.ts

```diff
@@ -3,7 +3,7 @@ import { untilDocumentReady } from '../../../utils/dom'
 import { getUrl, downloadUrl, pasteImageToActiveElements } from '../../../utils/utils'
 import Services from '../../../extension/service'
 import { decodeArrayBuffer } from '../../../utils/type-transform/String-ArrayBuffer'
-import { GrayscaleAlgorithm } from '@dimensiondev/stego-js/es/grayscale'
+import { GrayscaleAlgorithm } from '@dimensiondev/stego-js/cjs/grayscale'
```

Comment:
What about these changes? Use both es and cjs version are not cool

Reply:
```
@dimensiondev/stego-js/esm
```

compiler module is `es6`

Reply:
orignal `es` directory, output is `commonjs`
current `cjs` directory, output `commonjs`, `esm` directory output `esmodule`